src/components/Modals/AddContactModal.jsx

```diff
@@ -151,39 +163,65 @@ const AddContactModal = ({ addContact, showAddContactModal, setShowAddContactMod
               )
             }}
           />
-          <DialogActions sx={{ width: '100%' }}>
-            <Box
-              sx={{
-                display: 'flex',
-                flexDirection: isSmallScreen ? 'column' : 'row',
-                gap: isSmallScreen ? '10px' : '8px',
-                justifyContent: 'center',
-                alignItems: 'center',
-                width: '100%'
-              }}
+          <TextField
+            margin="normal"
+            id="pod"
+            name="addPod"
+            label="Pod URL"
+            autoComplete="pod"
+            value={pod}
+            onChange={(e) => setPod(e.target.value)}
+            fullWidth
+          />
+          <InputLabel id="relationship-label">Relationship</InputLabel>
+          <Select
+            labelId="relationship-label"
+            id="relationship"
+            name="relationship"
+            value={relationship}
+            label="Relationship"
+            onChange={(e) => setRelationship(e.target.value)}
+            fullWidth
+          >
+            <MenuItem value={RELATIONSHIPS.Blank}></MenuItem>
```

Comment:
In React, you could close empty components without closing tags. But you'll need to include a `/` before the component to signify that it's closed, which is why the Lint is failing this line.

For example using what is shown here, the correct syntax should be `<MenuItem value={RELATIONSHIPS.Blank} />` and that should be fine.

As for what I think about adding these new inputs, see feedback from before.